de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/transfer/HUTransformService.java

```diff
@@ -288,7 +288,7 @@ private IHUProductStorage getSingleProductStorage(@NonNull final I_M_HU cuHU)
 	 * @param qtyCU the CU-quantity to join or split
 	 * @param targetTuHU the target TU
 	 */
-	public void cuToExistingTU(
+	public List<I_M_HU> cuToExistingTU(
```

Comment:
pls document which is the return value because it's not obvious

Reply:
done